### PR TITLE
Added some documentation

### PR DESCRIPTION
**Pull request by yoshutch**
First stab at documenting this pipeline module.
Also don't error out if there are not .tfvars files in the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,18 +25,26 @@ module "codepipeline" {
...
-| repo_name | string | The name of the repository of the project. |
-| branch | string | The name of the branch you want to trigger the pipeline. |
-| deploy_provider | string | The provider for the deploy stage of the pipeline. If set to null this will skip setting up a deploy phase. | null |
-| deploy_configuration | any | The configuration for the deploy provider. If set to null this will skip setting up a deploy phase. | null |
-| build_buildspec | string | The file to use for the build phase. | buildspec.yml |
-| role_permissions_boundary_arn | string | The role permissions boundary ARN. |
-| github_token | string | The GitHub token associated with the AWS account. |
-| power_builder_role_arn | string | The ARN for the PowerBuilder role. |
+| pipeline_name | string | Unique name for the pipeline. No spaces. | |
+| acs_env | string | Environment of the AWS Account (e.g. dev, prd) | |
```
Comment (jvisker) [inline README.md]: Would you remind removing `acs_env` this and the variable. It isn't used now.

File: variables.tf [hunk 2]
```diff
@@ -23,6 +23,7 @@ variable "power_builder_role_arn" {
 //Source
 variable "source_github_owner" {
   type    = string
+  description = "The GitHub owner of the GitHub repo (the GitHub org or individual)."
   default = "byu-oit"
```
Comment (jvisker) [inline variables.tf]: Should we default the owner?

File: README.md (hunk 1 above)
Comment (yoshutch) [reply README.md]: removed

File: variables.tf (hunk 2 above)
Comment (yoshutch) [reply variables.tf]: removed

**Review by jvisker — APPROVED** [review]

**Pull request merged**